deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/charts/CurrenciesChart.vue

```diff
@@ -109,6 +109,7 @@ export default {
     this.$tokenMetricService.getMetrics()
```

Comment:
it would be better to move this to parent

Reply:
done